src/components/LogoutButton.js

```diff
@@ -3,8 +3,10 @@ import { makeStyles } from "@material-ui/core/styles";
 import { useDispatch } from "react-redux";
 import { IconButton } from "@material-ui/core";
 import { ExitToApp } from "@material-ui/icons";
-import { logout } from "../actions";
+import {baseApiUrl, logout} from "../actions";
 import { useHistory } from "../helpers/history";
+import {useModulesManager} from "../helpers/modules";
+import {M_PASS_LOGOUT_PATH} from "../constants";
```

Comment:
Lack of space before and after imported stuff. Btw, do you have configured prettify? It'll format this automatically.
`
import { baseApiUrl, logout } from "../actions";
import { useModulesManager } from "../helpers/modules";
import { M_PASS_LOGOUT_PATH } from "../constants";`

Reply:
done